src/bin/ion/commands/head.rs

```diff
@@ -53,7 +53,14 @@ impl IonCliCommand for HeadCommand {
             let mut reader = Reader::new(AnyEncoding, input.into_source())?;
             let encoding = *output.encoding();
             let format = *output.format();
-            write_n_as(&mut reader, output, encoding, format, num_values)?;
+            write_n_as(
+                &mut reader,
+                output,
+                encoding,
+                format,
+                num_values,
+                None::<fn(Element) -> Result<Element>>,
```

Comment:
Extract a constant or variable for this one and it will probably put the function invocation back on one line.